Resources/Locale/en-US/job/job.ftl

```diff
@@ -3,3 +3,4 @@ job-greet-important-disconnect-admin-notify = You are playing a job that is impo
 job-greet-supervisors-warning = As the {$jobName} you answer directly to {$supervisors}. Special circumstances may change this.
 job-greet-join-notify-crew = { CAPITALIZE($jobName)} {$characterName} on deck!
 job-greet-join-notify-crew-announcer = Station
+job-greet-crew-shortages = Due to crew shortages, you have been granted extra access levels for this shift.
```

Comment:
```suggestion
job-greet-crew-shortages = As this station was initially staffed with a skeleton crew, additional access may have been added to your ID card.
```

skeleton message or riot

Reply:
The message only gets sent if your job has extended access, so no "may" is necessary.